journals/apps/journals/models.py

```diff
@@ -330,8 +330,14 @@ def update_service(client, data, service_name):
             except HttpNotFoundError as err:
                 # Only a WARN because this will often happen on JournalAboutPage creation.
                 logging.warn(f"JournalAboutPage unable to update {service_name} because UUID doesn't exist: {err.content}")
+            except HttpServerError as err:
+                # Only a WARN if ecommerce as this will happen on initial create of Journal
+                if service_name == 'ecommerce':
+                    logging.warn(f"JournalAboutPage unable to update {service_name} because UUID doesn't exist: {err}")
```

Comment:
Should this be `err.content` like the others?

Reply:
err.content in this case spews pages of html error content, perhaps the page that is shown in django when there is an exception